### PR TITLE
Add links

### PR DESCRIPTION
**Pull request by kamicut**
@danielfdsilva @smit1678 I think this is missing the license at the bottom of the README. Is it BSD-3-Clause?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,33 +28,15 @@ To set up the development environment for this website, you'll need to install t
...
+### Getting started
 
-```
-$ npm run lint
-```
-Lints the app according with the defined style.
+#### Environment Variables
 
-## Configuration
 The configuration options are defined in `app/assets/scripts/config.js`.
```
Comment (danielfdsilva) [inline README.md]: It's not really in `config.js` but in the appropriate file in the `config` folder. Should we mention this or the notes on that file are enough?

**Comment by danielfdsilva** [comment]
@kamicut don't know about the license. Which one do we add here @smit1678 ? it's missing from the `package.json` as well

File: README.md (hunk 1 above)
Comment (kamicut) [inline README.md]: I added local.js instead

**Comment by smit1678** [comment]
👍 to bsd-3

**Comment by smit1678** [comment]
@danielfdsilva @kamicut anything left here?

**Comment by danielfdsilva** [comment]
@smit1678 License was still missing and fixing merge conflicts. All done now

**Pull request merged**